docs/interplanetary-file-system.md

```diff
@@ -0,0 +1,67 @@
+---
+id: interplanetary-file-system
+title: IPFS (Interplanetary File System)
+sidebar_label: IPFS (Interplanetary File System)
+---
+
+The IPFS (Interplanetary File System) is a decentralized network protocol that is based on a distributed network, this to create a direct relationship with the p2p method ([peer to peer](https://es.wikipedia.org/wiki/Peer-to-peer)), the purpose of this protocol is to store files or information quickly and safely. In this way, the IPFS changes the traditional form of information storage from centralized to decentralized, bringing with it many benefits such as access to information without depending on a single storage point for its respective access, increased storage, control of use of your own data and among many other benefits.
```

Comment:
Remove "The" at the start of the sentence

Reply:
refer to IPFS as IPFS not "the IPFS"

In this way, the IPFS...

Reply:
include a link to the main IPFS page  

https://ipfs.io/

Reply:
Attended.